src/js/views/awardsRecords.js

```diff
@@ -120,7 +105,7 @@ function getPlayerAwards(p, awardType) {
     const formatYear = year => {
         return Object.keys(year).map(k => {
             const years = helpers.yearRanges(year[k].map(y => y.season)).join(', ');
-            return `${k} <small>(${years})</small>`;
+            return `${k} (${years})`;
```

Comment:
>Change it to:
 return <span>{k} <small>({years})</small></span>;
and import React in this file.

Ok, that was what I originally tried but I just kept getting {object} printing on the grid so I had something wrong somewhere apparently.